build/plugins/rollup-plugin-inline-fs.js

```diff
@@ -4,6 +4,8 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
+// @ts-nocheck - rollup is no longer a dev dep.
```

Comment:
Do we still use this file anywhere?

Reply:
The file will exist in git history if we want to dig it up. If we aren't using it I would rather just remove it.